service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfig.java

```diff
@@ -109,22 +109,23 @@ public HttpVersion getHttpVersion() {
     return HttpVersion.valueOf(property.get());
   }
 
-  public int getWorkerPoolSize() {
-    String workerPoolSizeKey = "servicecomb.service.registry.client.workerPoolSize";
+  public int getInstances() {
+    String instances = "servicecomb.service.registry.client.instances ";
```

Comment:
Define a constant is better

Reply:
There is a redundant space char at the tail of this string